### DISCUSSION TITLE
Issues committing to new restraint repo

### DISCUSSION THREAD

**Discussion by cbouchar** [restraint-harness/restraint#252] [General]
@StykMartin @renanrodrigo 
I committed the following changeset, however, I'm not able to assign it to myself or select reviewers.
It seems only Renan and Martin are owners so not a lot I can investigate with respect to repo config.

https://github.com/restraint-harness/restraint/pull/251

**Comment by renanrodrigo** [comment]
Just made you an Owner, @cbouchar - check again, it should be working now

**Reply by cbouchar** [reply]
Thank you Renan

**Reply by cbouchar** [reply]
Just checked it out.  Everything is back to what I'm accustomed to.
